resources/public/api/conf/3.0/schemas/retrieve/metadata.json

```
@@ -1,71 +1,153 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
-  "type": "object",
-  "properties": {
-    "calculationId": {
-      "description": "Object containing top-level metadata information about the retrieved tax calculation",
-      "type": "string"
+  "metadata": {
+    "description": "Object containing top-level metadata information about the retrieved tax calculation.",
+    "type": "object",
+    "properties": {
+      "calculationId": {
+        "description": "The unique identifier of the calculation. Must conform to the regular expression:",
```

"Must conform to the regular expression:" is automatically rendered by the "pattern" so don't need it here:
```suggestion
        "description": "The unique identifier of the calculation.",
```